### PR TITLE
Update readme with how to release

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,3 +150,8 @@ To learn more about this, see the [`optimade-python-tools`](https://github.com/M
 
 **Q: Why create an individual `config.json` file instead of just mounting an existing `.aiida` directory and using that directly?**  
 **A:** This, currently, wouldn't work because the `REPOSITORY_URI` needs to point to the right path *inside* the container, not on the host. Furthermore, storing all configurations in the same file can be fragile.
+
+## For mantainers
+
+To release the new version, go to GitHub release API of the repo create a new release and update the release information. 
+The release action will be triggered by newly created release.
```
Comment (CasperWA) [inline README.md:157]:
```suggestion
The release action will be triggered by newly created release.

Note, the tag should start with a `v` and be followed by a full semantic version (see [SemVer](https://semver.org)).
For example: `v2.3.12`.
```

**Review by CasperWA — APPROVED** [review]
Looks fine to me. I added a small comment concerning the tag naming convention, as this will be used to update the codebase with the new version.

File: README.md (hunk 1 above)
Comment (unkcpz) [reply README.md:157]: Thanks! Added.

File: README.md [hunk 2]
```diff
@@ -150,3 +150,10 @@ To learn more about this, see the [`optimade-python-tools`](https://github.com/M
 
 **Q: Why create an individual `config.json` file instead of just mounting an existing `.aiida` directory and using that directly?**  
 **A:** This, currently, wouldn't work because the `REPOSITORY_URI` needs to point to the right path *inside* the container, not on the host. Furthermore, storing all configurations in the same file can be fragile.
+
+## For mantainers
```
Comment (eimrek) [inline README.md]:
```suggestion
## For maintainers
```

Comment (unkcpz) [reply README.md]: thanks a lot! @eimrek

**Pull request merged**